test/e2e/benchmark/throughput.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/v2/pkg/appconsts"
+	"github.com/celestiaorg/celestia-app/v2/test/e2e/testnets"
+	"github.com/celestiaorg/celestia-app/v2/test/util/testnode"
+)
+
+const seed = 42
+
+func main() {
+	if err := E2EThroughput(); err != nil {
+		log.Fatalf("--- ERROR Throughput test: %v", err.Error())
+	}
+}
+
+func E2EThroughput() error {
+	os.Setenv("KNUU_NAMESPACE", "test")
+
+	latestVersion, err := testnets.GetLatestVersion()
+	testnets.NoError("failed to get latest version", err)
+
+	log.Println("=== RUN E2EThroughput", "version:", latestVersion)
+
+	// create a new testnet
+	testnet, err := testnets.New("E2EThroughput", seed, testnets.GetGrafanaInfoFromEnvVar())
+	testnets.NoError("failed to create testnet", err)
+
+	log.Println("Cleaning up testnet")
+	defer testnet.Cleanup()
+
+	// add 2 validators
+	testnets.NoError("failed to create genesis nodes", testnet.CreateGenesisNodes(2, latestVersion, 10000000, 0, testnets.DefaultResources))
+
+	// obtain the GRPC endpoints of the validators
+	gRPCEndpoints, err := testnet.RemoteGRPCEndpoints()
+	testnets.NoError("failed to get validators GRPC endpoints", err)
+	log.Println("validators GRPC endpoints", gRPCEndpoints)
+
+	// create txsim nodes and point them to the validators
+	log.Println("Creating txsim nodes")
+	// version of the txsim docker image to be used
+	txsimVersion := "a92de72"
```

Comment:
[optional] this could be extracted to a constant outside the test like we do in some other tests.

Reply:
i agree, although i tried to keep to the scope of this issue and not change the original tests too much. I think we could do more work to make e2e tests generally better and more readable in another pr. 

I'm also not sure if we need to refactor performance/experimental tests since they're isolated and not even run as part of the e2e test suite.

Reply:
> [optional] this could be extracted to a constant outside the test like we do in some other tests.

There is an issue concerning the replacement of this hardcoded version: https://github.com/celestiaorg/celestia-app/issues/3288. However, I agree that in its current state, it could be replaced by a constant. As I am actively working on the throughput tests, I can address this in my upcoming PRs. I've created a tracking issue for now to keep it on the radar: https://github.com/celestiaorg/celestia-app/issues/3343.